andela/ah-backend-iroquois
# feat(Preference): Users should be able to like and dislike articles

**What does this PR do?** 
   - It enables users to like and dislike articles.

**Description of Tasks to be completed?**
Have the following endpoints working

`POST /api/articles/`
 `Get /api/articles/`
`POST /api/articles/<slug>/like/`
`POST /api/articles/<slug>/dislike/`

**How should this be manually tested?**
- After cloning the repo, cd into the directory. 
- Run ```pip install -r requirements.txt```
- Use Postman to try out the endpoints.

Only authenticated users are allowed to access the above end points. So supply the token in the header 
- To like or dislike an article try the following end points. Remember to supply the article's slug in the URL.

`POST /api/articles/<slug>/like/`
`POST /api/articles/<slug>/dislike/`

- Now confirm that it working by listing these article. Take a look at the likes / dislike count
`POST /api/articles/`

**What are the relevant pivotal tracker stories?**

- [ #159952009](https://www.pivotaltracker.com/story/show/159952009)
